src/main/java/io/candydoc/ddd/extract_domain_vocabulary/ReflectionsDomainVocabularyFinder.java

```diff
@@ -5,16 +5,16 @@
 import io.candydoc.ddd.model.*;
 import io.candydoc.ddd.strategic_pattern.StrategicPattern;
 import io.candydoc.ddd.tactical_pattern.TacticalPattern;
+import io.github.classgraph.ClassGraph;
+import io.github.classgraph.ClassInfo;
+import io.github.classgraph.PackageInfoList;
+import io.github.classgraph.ScanResult;
 import java.lang.annotation.Annotation;
 import java.lang.reflect.Field;
 import java.util.*;
 import java.util.function.Function;
 import java.util.stream.Collectors;
 import lombok.extern.slf4j.Slf4j;
-import org.reflections8.Reflections;
-import org.reflections8.scanners.SubTypesScanner;
-import org.reflections8.util.ConfigurationBuilder;
-import org.reflections8.util.FilterBuilder;
 
 @Slf4j
 public class ReflectionsDomainVocabularyFinder implements DomainVocabularyFinder {
```

Comment:
Ta classe qui implémente `DomainVocabularyFinder` ne s'appelle plus`ReflectionsDomainVocabularyFinder` vu que tu n'utilises plus Reflections

Reply:
Actually, we're not using classGraph anymore so we kept this class name.